src/store/metadata/stub/content/editions/adult-obesity-prevalence-aged-18-years-and-over_2023-05.json

```diff
@@ -0,0 +1,155 @@
+        "issued": "2023-05-17T12:41:00Z",
+        "modified": "2023-12-13T09:01:00Z",
+        "next_release": "1900-01-01T00:00:00Z",
+        "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+        "creator": "https://staging.idpd.uk/publishers/office-for-health-improvement-and-disparities",
+        "contact_point": {
+          "name": "Office for Health Improvement and Disparities",
+          "email": "mailto:dhsc.publicenquiries@dhsc.gov.uk"
+        },
+        "topics": [
+          "https://staging.idpd.uk/topics/health-and-social-care"
+        ],
+        "frequency": "irregular",
+        "keywords": [
+          "obesity",
+          "health",
+          "subnational",
+          "community",
+          "county"
+        ],
+        "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+      "spatial_resolution":["E0", "E1", "E9"],
+      "spatial_coverage": "E92000001",
+      "temporal_resolution": ["P1Y"],
+      "temporal_coverage": {
+        "start": "2015-11-16T00:00:00Z",
+        "end": "2021-11-16T00:00:00Z"
+       },
+        "versions_url": "https://staging.idpd.uk/datasets/adult-obesity-prevalence-aged-18-years-and-over/editions/2023-05/versions",
+        "versions": [
+          {
+            "@id": "https://staging.idpd.uk/datasets/adult-obesity-prevalence-aged-18-years-and-over/editions/2023-05/versions/1",
+            "issued": "2023-05-17T12:41:00Z"
+          }
+        ],
+        "table_schema": {
+          "columns": [
+            {
+                "name": "period",
+                "datatype": "string",
+                "titles": "gregorian-interval",
+                "description": "The time period associated with the observation."
+              },
+              {
+                "name": "areacd",
+                "datatype": "string",
+                "titles": "Statistical Geography",
+                "description": "The geography code associated with the observation."
+              },
+              {
+                "name": "areanm",
+                "datatype": "string",
+                "titles": "Area Name",
+                "description": "The name assiciated with The geography code "
+              },
+              {
+                "name": "geography",
+                "datatype": "string",
+                "titles": "Geography Type",
+                "description": "The categorisation of the geography code"
+              },
+              {
```

Comment:
Please delete the column mapping for: `variable_name,  indicator, mission, category`.
They are set to false in the json file on the family-ess repo:

https://github.com/GSS-Cogs/family-ess/blob/main/adult-obesity-prevalence-aged-18-years-and-over/adult-obesity-prevalence-aged-18-years-and-over.json